.travis.yml

```diff
@@ -38,4 +38,4 @@ install:
     - if [ "$TRAVIS_PHP_VERSION" = "hhvm" ]; then composer remove --dev --no-update kriswallsmith/spork; fi;
     - composer update $COMPOSER_FLAGS
 
-script: phpunit -v
+script: vendor/bin/phpunit -v
```

Comment:
To my surprise all checks have passed after I moved it back (with only PHPUnit warnings).